content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+
+You can reload logs from specific time ranges and query filters to meet specific needs. This document shows you how to configure and use this feature. 
+
+## Configuration
+
+In order to reload log events from your archives, Datadog uses the IAM Role in your AWS account that you configured for [your AWS integration][2]. If you have not yet created that Role, [follow these steps to do so][3]. To allow that Role to reload log events from your archives, add the following permission statement to its IAM policies (be sure to edit the bucket names and, if desired, specify the paths that contain your log archives):
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "DatadogUploadLogArchives",
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2_/_MY_OPTIONAL_BUCKET_PATH_2>/*"
+            ]
+        },
+        {
+            "Sid": "DatadogReloadLogArchivesListBucket",
+            "Effect": "Allow",
+            "Action": "s3:ListBucket",
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1>",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2>"
+            ]
+        },
+        {
+            "Sid": "DatadogReloadLogArchivesGetObject",
+            "Effect": "Allow",
+            "Action": "s3:GetObject",
+            "Resource": [
+                "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+                "arn:aws:s3:::<MY_BUCKET_NAME_2_/_MY_OPTIONAL_BUCKET_PATH_2>/*"
+            ]
+        }
+    ]
+}
+```
+
+## Updating Legacy S3 Archives for Reloading
+
+Originally, Datadog's AWS S3 archiving feature depended on your S3 bucket policy to grant appropriate write access to your S3 bucket (this is the industry standard for S3 log archival). But for added scurity, Datadog now writes to your AWS S3 log archives using the IAM Role in your AWS account that you configured for [your AWS integration][2]. 
+
+Datadog only supports reloading from archives that have been configured to use role delegation. Once you have modified your Datadog IAM role to include the IAM policy above, simply edit your archive in the Log Configuration page, select which AWS Account + Role combination you wish to write and reload your archives, and then save your archive. 
+
+## Starting an Archive Reload Job
+
+To start reloading log events from your S3 archives, go to the [Historical Views tab][4] of the [Logs Configuration page][5] in your Datadog account. Select [the "Reload from Archives" button][6]. 
```

Comment:
Let's remove the S3 reference as we will cover others in the future. We should just say `from your archives`. (IMHO)